app/assets/stylesheets/alchemy/selects.scss

```diff
@@ -17,7 +17,7 @@ select {
       @include button($base-color: $white, $padding: 0 5px);
       display: block;
       font-weight: normal;
-      border-color: $button-border-color;
```

Comment:
> Were there any specific button variables you had in mind to override the Bourbon defaults?

I'm now realizing you linked to these, so nevermind. I'll see about incorporating some and pushing up another commit.